src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -178,47 +179,16 @@ private sealed class UncommonFields
             public ParameterSymbol _lazyThisParameter;
             public Tuple<CultureInfo, string> _lazyDocComment;
             public OverriddenOrHiddenMembersResult _lazyOverriddenOrHiddenMembersResult;
-            public ImmutableArray<CSharpAttributeData> _lazyCustomAttributes;
-            public ImmutableArray<string> _lazyConditionalAttributeSymbols;
+            public ImmutableArray<CSharpAttributeData> _lazyCustomAttributes = ImmutableArray<CSharpAttributeData>.Empty;
+            public ImmutableArray<string> _lazyConditionalAttributeSymbols = ImmutableArray<string>.Empty;
             public ObsoleteAttributeData _lazyObsoleteAttributeData;
             public DiagnosticInfo _lazyUseSiteDiagnostic;
         }
 
-        private UncommonFields CreateUncommonFields()
-        {
-            var retVal = new UncommonFields();
-            if (!_packedFlags.IsObsoleteAttributePopulated)
-            {
-                retVal._lazyObsoleteAttributeData = ObsoleteAttributeData.Uninitialized;
-            }
-
-            if (!_packedFlags.IsUseSiteDiagnosticPopulated)
-            {
-                retVal._lazyUseSiteDiagnostic = CSDiagnosticInfo.EmptyErrorInfo; // Indicates unknown state.
-            }
-
-            if (_packedFlags.IsCustomAttributesPopulated)
-            {
-                retVal._lazyCustomAttributes = ImmutableArray<CSharpAttributeData>.Empty;
-            }
-
-            if (_packedFlags.IsConditionalPopulated)
-            {
-                retVal._lazyConditionalAttributeSymbols = ImmutableArray<string>.Empty;
-            }
-
-            if (_packedFlags.IsOverriddenOrHiddenMembersPopulated)
-            {
-                retVal._lazyOverriddenOrHiddenMembersResult = OverriddenOrHiddenMembersResult.Empty;
-            }
-
-            return retVal;
-        }
-
         private UncommonFields AccessUncommonFields()
         {
             var retVal = _uncommonFields;
-            return retVal ?? InterlockedOperations.Initialize(ref _uncommonFields, CreateUncommonFields());
+            return retVal ?? InterlockedOperations.Initialize(ref _uncommonFields, new UncommonFields());
```

Comment:
Note to myself: add a comment about UncommonFields intentionally being zero-inited. Per CR for the [https://github.com/dotnet/roslyn/pull/20497](https://github.com/dotnet/roslyn/pull/20497)